app/lib/persistenceGitbase/useGitbaseChatHistory.ts

```diff
@@ -61,11 +61,21 @@ export function useGitbaseChatHistory(options: RepoChatsOptions = {}) {
   const [files, setFiles] = useState<FileMap>({});
   const [loaded, setLoaded] = useState(false);
   const [loading, setLoading] = useState(false);
+  const [loadedFiles, setLoadedFiles] = useState(false);
```

Comment:
IMHO, `filesLoaded` seems more natural than `loadedFiles` for representing `bool` state.